js/id/ui/splash.js

```diff
@@ -23,8 +23,8 @@ iD.ui.Splash = function(context) {
             .append('p')
             .html(t('splash.text', {
                 version: iD.version,
-                website: '<a href="http://ideditor.com/">ideditor.com</a>',
-                github: '<a href="https://github.com/openstreetmap/iD">github.com</a>'
+                website: '<a href="//ideditor.com/">ideditor.com</a>',
```

Comment:
This one isn't available over https
(we should really just move it to github pages)